modules/core/client/directives/pagination.client.directive.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+//Pagination Directive
+angular.module('core').directive('meanPagination', function () {
+    return {
+        restrict: 'AEC',
+        scope: {
+            meanPaginationData: '=meanPaginationData',
+            meanPaginationItemCount: '=meanPaginationItemCount',
+            meanPaginationUrl: '@meanPaginationUrl',
+            meanPaginationSearch: '@meanPaginationSearch'
+        },
+        controller: ['$scope', '$filter', function($scope, $filter) {
+
+            $scope.buildPager = function () {
+                $scope.pagedItems = [];
+                $scope.itemsPerPage = $scope.meanPaginationItemCount;
+                $scope.currentPage = 1;
```

Comment:
Doesn't 0-index make the code easier? We would just do `{{currentPage + 1}}`, which I think more people are used to.